rl_chain/slates_chain.py

```diff
@@ -268,7 +266,7 @@ def _call_after_scoring_before_learning(
         self, event: Event, response_quality: Optional[float]
     ) -> SlatesPersonalizerChain.Event:
         event.selected.score = response_quality
-        self._reward.append(response_quality)
+        self.metrics.on_feedback(response_quality)
```

Comment:
instead of calling this in the separate chains, it feels like it should just be called 1. after _call does the scoring and 2. in delayed_score function

Reply:
yeah maybe